source.mak

```diff
@@ -370,7 +370,7 @@ PARSER_SRCS =				\
 	parsers/lua.c			\
 	parsers/m4.c			\
 	parsers/make.c			\
-	parsers/markdown.c			\
+	parsers/markdown.c		\
```

Comment:
Let's remove this change from this pull request; this is nothing to do with V.

Reply:
Genuinely don't know how that got in there! Removed.